tests/string.test.js

```diff
@@ -0,0 +1,17 @@
+const stringLength = require('../functions/strings');
+
+test('string length is 10', () => {
+  expect(stringLength('abcdefghij')).toBe(10);
+});
+
+test('string length is 3', () => {
+  expect(stringLength('abc')).toBe(3);
+});
+
+test('string leght is 6', () => {
+  expect(stringLength('123456')).toBe(6);
+});
+
+test('Expect that this test shows an error', () => {
+  expect(stringLength('mrEckendonk')).toBe(11);
+});
```

Comment:
This test function is failing on my end. We should test if the function is returning an error and if it does it should pass the test.
You could use .toThrow() instead of .toBe() that way it'll pass the test while function returns an error.

Reply:
😂 It gave an error when I ran the test and thought I did a great job because the string.test.js gave an error. Now I get the picture. Didn't realize the screenshot below was wrong

![image](https://user-images.githubusercontent.com/30936842/178682004-6721d187-6236-44fa-8588-93559f792ad9.png)

Reply:
![image](https://user-images.githubusercontent.com/30936842/178684925-03668e8d-79cf-40a7-8fb0-7ed65d3f95b9.png)